DEPRECATIONS.md

```diff
@@ -0,0 +1,5 @@
+# Deprecated APIs
+
+### DEP0001
+
+`hops-config` is deprecated. Please use the [`withConfig()`](https://github.com/xing/hops#withconfigcomponent) HOC from `hops` instead.
```

Comment:
Should we instead point to the `ConfigContext` as that would also work with hooks?